packages/cli/src/cmds/validator/voluntaryExit.ts

```diff
@@ -137,11 +156,12 @@ ${validatorsToExit.map((v) => `${v.pubkey} ${v.index} ${v.status}`).join("\n")}`
   },
 };
 
-async function processVoluntaryExit(
```

Comment:
I would instead split this into a `signVoluntaryExit` fn (rather than a `signAndMaybe...`, then the caller can chose to either submit or save to disk